packages/forms-web-app/src/dynamic-forms/dynamic-components/multi-file-upload/question.js

```diff
@@ -166,13 +168,14 @@ class MultiFileUploadQuestion extends Question {
 		}
 	};
 
-	async #saveFilesToBlobStorage(req, journeyResponse) {
+	async #saveFilesToBlobStorage(req, journeyResponse, errors) {
```

Comment:
part of me thinks it would be more logical to pass through valid files rather than errors to this function but that's a little nit picky

Reply:
done